README.md

```diff
@@ -80,3 +80,13 @@ If you get an `ImportError` from python, you may need to set the `PYTHONPATH` en
 ```bash
 export PYTHONPATH=/path/to/venv/lib/python3.x/site-packages
 ```
+
+When submitting a PR we check coverage. You can check coverage locally with
+
+```bash
+$ cargo llvm-cov --workspace
```

Comment:
What does `--workspace` add?

Reply:
Actually looks like you can use `cargo llvm-cov` only. But for the record, `--workspace` would test all packages in the workspace.